lib/FCM/CLI/Parser.pm

```diff
@@ -66,7 +66,7 @@ our %OPTION_OF = map {
     [['jobs'               ,            ], ['j'], OPT_SCAL],
     [['list'               ,            ], ['l'], OPT_BOOL],
     [['name'               ,            ], ['n'], OPT_SCAL],
-    [['new'                ,            ], ['n'], OPT_BOOL],
```

Comment:
I guess that because the `-n` option was undocumented, changing it won't matter.